denis.roman/task2.rb

```diff
@@ -0,0 +1,2 @@
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+p hash.count { |k, _| k.to_s.include?('yes') }
```

Comment:
#include? is not the case here. include will check if String include? given word. You need to find a better method to use here.